custom/icds_reports/templates/icds_reports/icds_app/pdf/issnip_monthly_register.html

```diff
@@ -58,7 +63,7 @@ <h2>1. Basic Details</h2>
                                     <td>*Name Of Supervisor: </td>
                                 </tr>
                                 <tr>
-                                    <td>Name of AWW: {{ report.agg_awc_monthly_data|icds_format:"aww_name" }}</td>
+                                    <td>Name of AWW: <span style="font-family: HindGuntur-Regular">{{ report.agg_awc_monthly_data|icds_format:"aww_name" }}</span></td>
```

Comment:
Does this affect outputs in latin characters as well?

Reply:
Yes, yet this font has Latin characters as well in it.